### PR TITLE
Contact me page

### PR DESCRIPTION
**Pull request by Laylo309**
Created contact me page
In index.html:
- created form 
- used formspree.io.
In styles.css:
- created style for the last page
- added images

### CODE REVIEW

**Comment by mikenath223** [comment]
Hi , 

You made a great effort in this project, but unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that will be invalidated for this reason:
-  Kindly add a description to the Pull Request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Happy coding. Cheers!🚀🚀

If something isn't 100% clear, you can leave comments or questions in this PR thread.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **don't count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -209,6 +209,35 @@ <h6 class="social-media-text">LET’S CONNECT</h6>
...
+            </li>
+
+            <li class="textarea">
+              <label for="msg"></label>
+              <textarea id="msg" name="user_message" placeholder="Write your message here" maxlength="500"
+                required></textarea>
+            </li>
+
+          </ul>
+          <button class="button" type="submit">Get in touch</button>
```
Comment (vichuge) [inline index.html:236]: - [ ] When I press the button, this doesn't change the color as the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) requires. Please, consider changing the color on the button when this is pressed on the form. 🐶 
image

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comment under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Laylo309) [reply index.html:236]: I made it enabled as Figma design should I use hover in this case. I think I have used the proper attribute.

Comment (carlos-ssh) [reply index.html:236]: - Kindly follow the requirements of your last reviewer. Isn't working that effect running locally or in your live version.
- Add .vscode to your .gitignore file.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Laylooo , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**